lib/Attribute.js

```diff
@@ -196,42 +197,54 @@ Attribute.prototype.types = {
   }
 };
 
+// This function takes in a `value` as a parameter and sets the type of this attribute based on the type of that value.
+// No type defined - assume this is not a type definition and we must grab type directly from value.
+// Examples of `value` parameters that might be passed in:
+// { type: [Function: String], required: true, default: 'Mittens' }
+// function String() { [native code] }
+// { type: [Function: Number], validate: [Function: validate], default: 888 }
+// 1
+// false
+// "puddles"
+// [ 'cute', 'fuzzy' ]
+// { name: 'mittens', age: 1 }
```

Comment:
Consider switching this doc over to the jsDoc standard.

[the standard](http://usejsdoc.org/about-getting-started.html)

Reply:
@hweeks This is an internal method. I'm not sure how that works with jsdoc. Users of Dynamoose shouldn't need to use this at all.

Reply:
In the new doc standard, we auto build lib docs. Having the JSDoc comments makes it easier for other devs to onboard, and reduces our double documentation writing efforts.

Reply:
@hweeks That makes sense. But is there a way to designate that this is an internal method? I don't think it'd be valuable to have our documentation cluttered with a bunch of internal methods that users won't ever have to use. If you are a new user to Dynamoose, on-boarding should be as easy as possible. Having to read through documentation that is filled with a bunch of internal methods that you don't need is kinda a waste.

Reply:
yup, [inner](http://usejsdoc.org/tags-inner.html)

Reply:
@hweeks Check out my latest commit and let me know your thoughts.